.yarn/patches/@metamask-eth-keyring-controller-npm-13.0.1-06ff83faad.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/dist/KeyringController.js b/dist/KeyringController.js
+index 3644209391f2cdf7ff5e2bd8f94517bff13b0f47..ed63d0ba4cb56df9fe3eb6b0a274f696375dd9dc 100644
+--- a/dist/KeyringController.js
++++ b/dist/KeyringController.js
+@@ -259,7 +259,7 @@ class KeyringController extends events_1.EventEmitter {
+         if (!keyring.removeAccount) {
+             throw new Error(constants_1.KeyringControllerError.UnsupportedRemoveAccount);
+         }
+-        keyring.removeAccount(address);
++        await keyring.removeAccount(address);
```

Comment:
Two things we need to take into account when starting patches,

1. Add a comment directly in the patch to explain it's purpose
2. Create a PR to do this same code modification in the respective package and also add the link to the PR or commit to the patch

Reply:
Done, although this should be more explicit and applied to other patches as well.